### ISSUE TITLE
Decrease tests execution time

### ISSUE THREAD

**Issue by dsilvam** [dotCMS/core#22495]
Currently our tests last around 2:30 hours to finish execution and give developers feedback. The idea is to come up with options to decrease this time, with the objetive of getting PRs merged into master sooner. 

These are the actual times from a recent run (the four different tests sets run in parallel, shown aligned horizontally): 

<img width="675" alt="Screen Shot 2022-06-29 at 12 05 30" src="https://user-images.githubusercontent.com/1576794/176494824-40a38be6-e6c6-4445-90c6-e6b353694c07.png">

Options for Integration Tests: 
* Run the test cases in this test class in parallel: com.dotcms.content.elasticsearch.util.ESMappingUtilHelperTest. Total exec time is 36m56.84s, which is a a good ~25% of the total time. 

Options for Postman Tests: 
* Run collections in parallel. They are currently running sequentially.

**Comment by nollymar** [comment]
The ESMappingUtilHelperTest is taking so long because on each execution a new request to create an index is sent to ES (which is taking up to a minute each time). I was trying to find a way avoid it, but the main purpose of these tests is to verify that the ES mapping is set correctly during index creation. 

Therefore, the only alternative I see is to find a way to run them in parallel, which is a bit challenging while we keep using JUnit 4 and gradle.

**Comment by wezell** [comment]
Can we create 1 content type with ALL the fields and then create 1 index and test that all the field mappings are correct?

**Comment by nollymar** [comment]
I've refactored the test here: https://github.com/dotCMS/core/pull/22506
